src/routes/api/conversations/+server.ts

```diff
@@ -14,6 +16,8 @@ export async function GET({ locals }) {
 				model: 1,
 			})
 			.sort({ updatedAt: -1 })
+			.skip(300 * p)
+			.limit(300)
```

Comment:
```suggestion
			.skip(NUM_PER_PAGE * p)
			.limit(NUM_PER_PAGE)
```

and declating `const NUM_PER_PAGE = 300;` above would be more robust since `skip` & `limit` should be tied with one another through a variable

Reply:
Should we make it an env var? So it's also tied to the one in the load function ?

Reply:
up to you. For me, it is fine as long as it is a variable here